libraries/libswing/src/main/java/org/pentaho/reporting/libraries/designtime/swing/ColorCellRenderer.java

```diff
@@ -124,19 +124,9 @@ protected void paintComponent( final Graphics graphics ) {
       graphics.setColor( color );
       graphics.fillRect( 0, 0, getWidth(), getHeight() );
 
-      final int r = color.getRed();
-      final int g = color.getGreen();
-      final int b = color.getBlue();
-      int brightness = ( r > g ) ? r : g;
-      if ( b > brightness ) {
-        brightness = b;
-      }
-
-      if ( brightness > 167 ) {
-        setForeground( Color.black );
-      } else {
-        setForeground( Color.white );
-      }
+      setForeground(
+        ( ColorUtility.getBrightness( color ) > ColorUtility.BRIGHTNESS_THRESHOLD ) ? Color.BLACK : Color.WHITE
```

Comment:
The `( )` wrapping the condition can be omitted

Reply:
Done